homeassistant/components/fyta/strings.json

```diff
@@ -36,51 +36,50 @@
       "plant_status": {
         "name": "Plant state",
         "state": {
-          "too_low": "Too low",
-          "low": "Low",
-          "perfect": "Perfect",
-          "high": "High",
-          "too_high": "Too high"
+          "deleted": "Deleted",
```

Comment:
In a future PR it would make sense to automatically remove them I think

Reply:
You are right. As I understand, this state is only given for plants, that have been deleted in the app and therefore also cannot have an associated Beam. Currently, only plants with a Beam sensor are showed in HA. Therefore, only a logic has to be applied for the case the state changes to "deleted".